swan-pipeline/src/main/java/de/fraunhofer/iem/swan/features/code/MethodGetterOrSetterFeature.java

```diff
@@ -0,0 +1,143 @@
+package de.fraunhofer.iem.swan.features.code;
+
+import de.fraunhofer.iem.swan.data.Category;
+import de.fraunhofer.iem.swan.data.Method;
+import de.fraunhofer.iem.swan.features.code.type.WeightedFeature;
+import soot.*;
+import soot.jimple.*;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class MethodGetterOrSetterFeature extends WeightedFeature implements IFeatureNew{
```

Comment:
WWe could merge this one with the constructor feature. It would then be MethodTypefeature (constructor, getter, setter, other)